views/tos.ejs

```diff
@@ -18,6 +18,9 @@
     <ul>
         <li>You allow your IP to be logged whilst making an account (This is so we can verify you're not
             multi-accounting)</li>
+        <li>You may not paste anything containing personal information, if you want to share personal info don't do it
```

Comment:
you can do that, just not others information and or child porn

Reply:
Alright yea lol